lib/vagrant-mutate/converter/kvm.rb

```diff
@@ -43,12 +43,27 @@ def write_specific_files
       private
 
       def find_kvm
-        qemu_bin_list = ['/usr/bin/qemu-system-x86_64',
-                         '/usr/bin/qemu-system-i386',
-                         '/usr/bin/qemu-kvm',
-                         '/usr/libexec/qemu-kvm',
-                         '/usr/bin/kvm']
-        qemu_bin = qemu_bin_list.find { |binary| File.exist? binary }
+        qemu_bin = nil
+        found = false
+
+        qemu_bin_list = ['qemu-system-x86_64',
+                         'qemu-system-i386',
+                         'qemu-kvm',
+                         'kvm']
+        qemu_bin_list.each do |qemu|
```

Comment:
There is already some path searching code at https://github.com/sciurus/vagrant-mutate/blob/a948b80f1b332e7c59cc41eec0feb3b49cd4d508/lib/vagrant-mutate/qemu.rb#L4-L18

Do you think you could extract a utility method that would work for both and use it there and here?